SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -452,7 +458,9 @@ void USpatialNetDriver::OnAcceptingPlayersChanged(bool bAcceptingPlayers)
 	if (bWaitingForAcceptingPlayersToSpawn && bAcceptingPlayers)
 	{
 		// If we have the correct map loaded then ask to spawn.
-		if (GetWorld() != nullptr && GetWorld()->RemovePIEPrefix(GlobalStateManager->DeploymentMapURL) == GetWorld()->RemovePIEPrefix(GetWorld()->URL.Map))
+		FString DeploymentMapURL = GlobalStateManager->GetDeploymentMapURL();
+		UWorld* World = GetWorld();
+		if (World && World->RemovePIEPrefix(DeploymentMapURL) == World->RemovePIEPrefix(World->URL.Map))
```

Comment:
We should be checking the SessionID here too no?

Reply:
✓